include/ppx/grfx/grfx_pipeline.h

```diff
@@ -172,6 +178,7 @@ struct GraphicsPipelineCreateInfo
     grfx::ColorBlendState          colorBlendState    = {};
     grfx::OutputState              outputState        = {};
     grfx::ShadingRateMode          shadingRateMode    = grfx::SHADING_RATE_NONE;
+    grfx::MultiViewState           multiViewMask      = {};
```

Comment:
Since it contains both view mask and correlation mask it should probably be named multiViewState or multiViewInfo